src/mesh/NekRSMesh.C

```diff
@@ -36,6 +36,7 @@ NekRSMesh::validParams()
   params.addParam<std::vector<int>>("boundary",
                                     "Boundary ID(s) through which nekRS will be coupled to MOOSE");
   params.addParam<bool>("volume", false, "Whether the nekRS volume will be coupled to MOOSE");
+  params.addParam<bool>("moving_mesh", false, "Whether we have a moving mesh problem or not");
```

Comment:
Could you please jog my memory on this - what was the motivation for having this `moving_mesh` parameter be a user-set quantity, if we could technically infer if the mesh is moving from the `.par` file settings? 

Is there ever a situation where we would have mesh deformation turned ON in the `.par` file, but have `moving_mesh = false` (or vice versa)? Just wondering if we can simply some of this logic by accessing information "at the root."

Reply:
I don't think we ever spent too much time thinking about this. We did this in summer '21, when nekRS didn't have the elasticity solver and we(mostly me) were still figuring things out.

I can't think of a single scenario where we won't be able to infer the desired setting from the `par` file.